inspec.yml

```diff
@@ -1,12 +1,117 @@
-# encoding: utf-8
-
+---
 name: windows-baseline
 title: DevSec Windows Security Baseline
+summary: An InSpec Compliance Profile for CIS Microsoft Windows Server 2012R2 and 2016 RTM (Release 1607) Benchmark Level 1 and 2 - Member Server and Domain Controller. It contains additional controls from the MS technet.
+version: 2.0.0
 maintainer: DevSec Hardening Framework Team
 copyright: DevSec Hardening Framework Team
 copyright_email: hello@dev-sec.io
-license: Apache 2 license
-summary: Baseline for best-practice Windows OS hardening
-version: 1.1.2
+license: Apache-2.0
 supports:
-  - os-family: windows
+- platform-family: windows
+attributes:
+  - name: level_1_or_2
+    required: false
+    description: 'define if you want to execute Level 1 or (Level 1 and Level 2)'
+    default: 1,
+    type: numeric
+  - name: ms_or_dc
+    required: false
+    description: 'define if you want to execute the profile in the context of a Memeber Server (MS) or Domain Controler (DC)'
+    default: 'MS'
```

Comment:
should we use lower-case here?

Reply:
i just followed the CIS docs the use MS for Member Server and so on